pkg/apiserver/bootstrap.go

```diff
@@ -0,0 +1,142 @@
+package apiserver
+
+import (
+	"context"
+
+	"github.com/cenkalti/backoff/v4"
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated-sdk/pkg/appstate"
+	appstatetypes "github.com/replicatedhq/replicated-sdk/pkg/appstate/types"
+	"github.com/replicatedhq/replicated-sdk/pkg/heartbeat"
+	"github.com/replicatedhq/replicated-sdk/pkg/helm"
+	"github.com/replicatedhq/replicated-sdk/pkg/k8sutil"
+	sdklicense "github.com/replicatedhq/replicated-sdk/pkg/license"
+	"github.com/replicatedhq/replicated-sdk/pkg/mock"
+	"github.com/replicatedhq/replicated-sdk/pkg/store"
+	"github.com/replicatedhq/replicated-sdk/pkg/upstream"
+	upstreamtypes "github.com/replicatedhq/replicated-sdk/pkg/upstream/types"
+	"github.com/replicatedhq/replicated-sdk/pkg/util"
+)
+
+func bootstrap(params APIServerParams) error {
+	verifiedLicense, err := sdklicense.VerifySignature(params.License)
+	if err != nil {
+		return backoff.Permanent(errors.Wrap(err, "failed to verify license signature"))
+	}
+
+	if !util.IsAirgap() {
+		// sync license
+		licenseData, err := sdklicense.GetLatestLicense(verifiedLicense, params.ReplicatedAppEndpoint)
+		if err != nil {
+			return errors.Wrap(err, "failed to get latest license")
+		}
+		verifiedLicense = licenseData.License
```

Comment:
was probably like this already, but should we verify the signature again here when we get the updated license? someone could mitm replicated.app and send back something invalid

Reply:
the SDK is hitting `https://replicated.app`, and i believe the client will bail and complain if it doesn't get a certificate that matches the replicated.app hostname?